_includes/draft-warning.html

```diff
@@ -4,6 +4,7 @@
     document.location.hostname === 'localhost'
     || document.location.hostname === 'solid.github.io'
     || document.location.hostname.substring(document.location.hostname.length - 10) === "vercel.app"
+    || document.location.hostname.substring(document.location.hostname.length - 10) === "netlify.app"
```

Comment:
Ah well-spotted! `netlify.app` is eleven characters though :)

```suggestion
    || document.location.hostname.substring(document.location.hostname.length - 11) === "netlify.app"
```